graphix/sim/tensornet.py

```diff
@@ -502,28 +504,30 @@ def get_basis_amplitude(self, basis, **kwagrs):
         coef = self.get_basis_coefficient(basis, **kwagrs)
         return abs(coef) ** 2
 
-    def to_statevector(self, indices=None, **kwagrs):
-        """Retrieve the statevector from the tensornetwork.
-        This method tends to be slow however we plan to parallelize this.
+    def to_statevector(self, skip=False, **kwagrs) -> NDArray:
+        """Take outer product of the tensors in the network and return the statevector.
 
         Parameters
         ----------
-        indices (optional): list of int
-            target qubit indices. Default is the MBQC output nodes (self.default_output_nodes).
+        skip : bool
```

Comment:
argument name `skip` is way too generic and very difficult to understand. Consider making it `skip_tn_simp` or restructure. Also, why/who/when do we want to skip the simplification?

Reply:
This option is not that essential, so I removed `skip`.